configure.ac

```diff
@@ -375,6 +376,22 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
 )
 CXXFLAGS="$TEMP_CXXFLAGS"
 
+TEMP_CXXFLAGS="$CXXFLAGS"
+CXXFLAGS="$TEMP_CXXFLAGS $POWER8_CXXFLAGS"
+AC_MSG_CHECKING(for POWER8 compiler support)
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
+    #include <altivec.h>
+    #include <stdint.h>
+  ]], [[
+    unsigned char src[16];
+    __builtin_crypto_vshasigmaw((__vector uint32_t)vec_vsx_ld(0, src), 1, 0xf);
```

Comment:
Better to use `vector` here. We don't use [the non-standard] `__vector` in the code itself.